1-chat-models/3-manual-memory/README.md

```diff
@@ -12,5 +12,8 @@ You need to set the `OPENAI_API_KEY` environment variable to the value of your O
 
 ## Running with local Ollama instead
 If you'd like to run with a local Ollama instance instead, first you need to ensure a local Ollama instance is running on port `11434` and the `llama3.2` model is pulled.
+* install ollama
+* `ollama serve`
+* (on another terminal) `ollama run llama3.2`
```

Comment:
These steps shouldn't actually be needed as Quarkus should ensure that Ollama is serving the proper model

Reply:
+1

Reply:
I think I had to do it but I'll need to double check

Reply:
@geoand I think it only does this if you are using the `quarkus-langchain4j-ollama` extension. Its that dev service which will pre-load the model.

Reply:
I mean the changes to the readme